packages/thirdweb/src/transaction/actions/send-transaction.ts

```diff
@@ -110,6 +110,32 @@ export interface SendTransactionOptions {
  * });
  * ```
  *
+ * ### Send an EIP-7702 Transaction
+ *
+ * **Note: This feature is in beta and is subject to breaking changes**
+ *
+ * ```ts
+ * import { sendTransaction, prepareTransaction } from "thirdweb";
+ * import { sepolia } from "thirdweb/chains";
+ *
+ * const transaction = prepareTransaction({
+ *   chain: sepolia,
+ *   client: client,
+ *   authorizations: [
```

Comment:
authorizationList or authorizations? i think it should be the former right?

Reply:
Fixed